src/backends/interflop-vprec/interflop_vprec.c

```diff
@@ -939,6 +1177,9 @@ static error_t parse_opt(int key, char *arg, struct argp_state *state) {
 static struct argp argp = {options, parse_opt, "", "", NULL, NULL, NULL};
 
 void init_context(t_context *ctx) {
+  ctx->relErr = true;
+  ctx->absErr = false;
+  ctx->absErr_exp = 112;
```

Comment:
Why 112 ? We need a comment here or a CONSTANT.

Reply:
I've replaced this initialization with the corresponding constant from float_const.h